packages/features/calendars/weeklyview/components/Calendar.tsx

```diff
@@ -28,26 +29,9 @@ export function Calendar(props: CalendarComponentProps) {
   const hideHeader = useCalendarStore((state) => state.hideHeader);
 
   const days = useMemo(() => getDaysBetweenDates(startDate, endDate), [startDate, endDate]);
-
```

Comment:
Deleted responsive behaviour for event blocks I added in an earlier PR. 
Before we made sure all hours fitted in the screen. Now we changed that to always showing 24 hours and a scroll.

Reply:
Nice i much prefer this approach